[2024-09-13T16:06:25.562+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-13T16:06:25.600+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_postgres.say_welcome_postgres scheduled__2024-09-09T00:00:00+00:00 [queued]>
[2024-09-13T16:06:25.624+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_postgres.say_welcome_postgres scheduled__2024-09-09T00:00:00+00:00 [queued]>
[2024-09-13T16:06:25.626+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-13T16:06:25.664+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): say_welcome_postgres> on 2024-09-09 00:00:00+00:00
[2024-09-13T16:06:25.676+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5923) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-13T16:06:25.678+0000] {standard_task_runner.py:72} INFO - Started process 5924 to run task
[2024-09-13T16:06:25.678+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_postgres', 'say_welcome_postgres', 'scheduled__2024-09-09T00:00:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/test_postgres_dags.py', '--cfg-path', '/tmp/tmpl2ihuzr_']
[2024-09-13T16:06:25.680+0000] {standard_task_runner.py:105} INFO - Job 104: Subtask say_welcome_postgres
[2024-09-13T16:06:25.782+0000] {task_command.py:467} INFO - Running <TaskInstance: test_postgres.say_welcome_postgres scheduled__2024-09-09T00:00:00+00:00 [running]> on host 9c5d837b6f3b
[2024-09-13T16:06:25.970+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='muhii.smta@gmail.com' AIRFLOW_CTX_DAG_OWNER='Muhii' AIRFLOW_CTX_DAG_ID='test_postgres' AIRFLOW_CTX_TASK_ID='say_welcome_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-09T00:00:00+00:00'
[2024-09-13T16:06:25.972+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-13T16:06:26.013+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-13T16:06:26.015+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hallo !"']
[2024-09-13T16:06:26.037+0000] {subprocess.py:86} INFO - Output:
[2024-09-13T16:06:26.044+0000] {subprocess.py:93} INFO - Hallo !
[2024-09-13T16:06:26.046+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-13T16:06:26.100+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-13T16:06:26.102+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_postgres, task_id=say_welcome_postgres, run_id=scheduled__2024-09-09T00:00:00+00:00, execution_date=20240909T000000, start_date=20240913T160625, end_date=20240913T160626
[2024-09-13T16:06:26.153+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-13T16:06:26.211+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-13T16:06:26.216+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
